build/products.yaml

```diff
@@ -4,7 +4,7 @@ PX-Enterprise:
   PRODUCT_NAME: Portworx Enterprise
   PRODUCT_INDEX_NAME: PX-Enterprise
   VERSIONS_BASE_URL: docs.portworx.com
-  OTHER_PRODUCT_NAMES_AND_INDICES: "PX-Backup=PX-Backup-2-2,PX-Central=PX-Central-2-2"
+  OTHER_PRODUCT_NAMES_AND_INDICES: "Portworx Backup=Portworx Backup-2-2,PX-Central=PX-Central-2-2"
```

Comment:
The values in `products.yaml` should stay the same. These can't include spaces and configure service, pod, and index values used in GKE and Algolia.

Reply:
Actually, I _think_ you can change `PRODUCT_NAME`, but you need to keep `PRODUCT_INDEX_NAME`. see Portworx enterprise for example.

Reply:
That's weird, I thought I had done that. Must've been a rogue find and replace.